concordia/tests/test_views.py

```diff
@@ -1088,6 +1088,59 @@ def test_duplicate_tag_submission(self):
         self.assertEqual(["bar", "foo", "quux"], data["user_tags"])
         self.assertEqual(["baaz", "bar", "foo", "quux"], data["all_tags"])
 
+    def test_tag_deletion(self):
+        asset = create_asset()
+        self.login_user()
+
+        test_tags = ["foo", "bar"]
+        resp = self.client.post(
+            reverse("submit-tags", kwargs={"asset_pk": asset.pk}),
+            data={"tags": test_tags},
+        )
+        test_tags = [
```

Comment:
I would use a different variable name to make it clear that you are intentionally removing only one of the values — “test_tags” is a very generic name so I'd probably go with something like “initial_tags” and “updated_tags” to make the subsequent assertions more explanatory.